libass/ass.c

```diff
@@ -629,6 +670,58 @@ static void event_format_fallback(ASS_Track *track)
             "No event format found, using fallback");
 }
 
+/**
+ * \brief Guess if track is a legacy conv. sub and set headers accordingly
+ * \param track track
+*/
+static void detect_legacy_conv_subs(ASS_Track *track)
+{
+    if(track->parser_priv->comp_header_set)
+        return;
+
+    if(track->track_type != TRACK_TYPE_ASS)
+        track->parser_priv->conv_score -= 5;
+
+    //2 not 1 because libass also adds a def style
+    if(track->n_styles != 2)
+        track->parser_priv->conv_score -= 3;
+    else if(!strncmp(track->styles[1].Name, "Default", 7))
+        track->parser_priv->conv_score += +3;
```

Comment:
Nit: is this unary plus intentional?

Reply:
thx, fixed this and other styling issues